components/event/eventSkeleton.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <article>
```

Comment:
Philosophical question but... do we really need skeleton with a `nuxt generate` site? 
Are we sure the skeleton code is not imported in the final generated file? 
CC @f3ltron

Reply:
And if it's not needed, I'd simply remove the code. Less code to manage and think about is always better.

Reply:
it give use a nice lazy load datas for each successive routes

Reply:
Ahh ok cool! Yeah I see it now if I put 3G slow!  Great, good thing to have those then!